internal/format/first_pass_visitor.go

```diff
@@ -39,10 +40,14 @@ type firstPassVisitor struct {
 	Imports []*proto.Import
 
 	haveHitNonComment bool
+
+	updateFileOptions bool
```

Comment:
nit: Maybe `fixFileOptions` or `repairFileOptions`?

Reply:
I want this to match the flag